diff --git a/Loop/WatchApp Extension/Scenes/GlucoseChartScene.swift b/Loop/WatchApp Extension/Scenes/GlucoseChartScene.swift
index 8f69ff5e..34bba671 100644
--- a/Loop/WatchApp Extension/Scenes/GlucoseChartScene.swift	
+++ b/Loop/WatchApp Extension/Scenes/GlucoseChartScene.swift	
@@ -297,7 +297,7 @@ class GlucoseChartScene: SKScene {
 
         data.historicalGlucose?.filter { scaler.dates.contains($0.startDate) }.forEach {
             let center = scaler.point($0.startDate, $0.quantity.doubleValue(for: unit))
-            let size = CGSize(width: 2, height: 2)
+            let size = CGSize(width: 3, height: 3)
             let origin = CGPoint(x: center.x - size.width / 2, y: center.y - size.height / 2)
             let (sprite, created) = getSprite(forHash: $0.chartHashValue)
             sprite.color = .glucose
